Client/Info.plist

```diff
@@ -113,6 +115,8 @@
 	<string>$(NIMBUS_URL)</string>
 	<key>PocketEnvironmentAPIKey</key>
 	<string>$(POCKET_API_KEY)</string>
+	<key>STAGING_BRAZE_API_KEY</key>
+	<string>$(STAGING_BRAZE_API_KEY)</string>
```

Comment:
How is this planned to be shared? I'm guessing on Bitwarden and then part of the `Staging.xcconfig` ignored by git?

And why is this only required for staging? Is this also a cloudflare zero trust environment?

Reply:
Actually, after double checking the core side PR I noticed it expects  a `"\(environmentPrefix)_\(keyApiKey)"` format, which means this should exist for both environments. Don't we need to add a `PRODUCTION` one here? Will this be done by CI somehow?

Reply:
Not done at the CI level, thanks for spotting it! This is something I missed in the `Info.plist` and needed to add 🙌 🙏

Reply:
I shared it on Bitwarden as soon as it was grabbed.
Also, the `Staging.xcconfig` is planned to be updated in the Env var already 💪 .
Core also already has the value grabbed by the Ci env vars.

Reply:
Makes sense, I guess this only applies for deploying staging to appcenter, doesn't it? Don't we need something like `Production.xcconfig` that is also added to CircleCI and contains Braze's prod api key so we can use it when build for release to testflight?

Reply:
💯 We do need `Production.xcconfig` as part of it yes.